tpm2/test/tpm2_test.go

```diff
@@ -1102,6 +1102,38 @@ func TestReadPCR(t *testing.T) {
 	}
 }
 
+func TestPCRReset(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+	allZeroBytes := make([]byte, 32)
+
+	var fakeHashSum [32]byte
+	err := PCRExtend(rw, 16, AlgSHA256, fakeHashSum[:], "")
```

Comment:
We should have PCR 16 defined as a constant here, instead of repeating over and over.